python/tvm/contrib/popen_pool.py

```diff
@@ -269,9 +269,14 @@ class PopenPoolExecutor:
 
     timeout : float
         Timeout value for each function submit.
+    Note
+    ----
+    If max_workers is NONE then the number returned by
+    os.cpu_count() is used. This method aligns with the
+    behavior of multiprocessing.pool().
     """
 
-    def __init__(self, max_workers, timeout=None):
+    def __init__(self, max_workers=os.cpu_count(), timeout=None):
```

Comment:
TVM has its own function to get number of threads. Should we just use that code instead?

Reply:
this is to make default behavior consistent with `multiprocessing.pool`

Reply:
I was thinking `multiprocessing.cpu_count()` but that is essentially a wrapper for `os.cpu_count()`.  Would that be possible that you can point me to the place with tvm's own function to get number of threads?

Reply:
Let's make sure the behavior is consistent in this PR.

Just a nitpick, I would prefer defaulting `max_worker=None`, and do the checks in the function body, so that the signature could be a tiny little bit easier to read:

```python
if max_workers is None:
  max_workers = os.cpu_count()
```

Reply:
done